tcs-service/src/test/java/com/transformuk/hee/tis/tcs/message/RevalidationMessageListenerTest.java

```diff
@@ -32,16 +30,24 @@ public class RevalidationMessageListenerTest {
   private final String NO_EXCHANGE = "false";
   private final String ROUTING_KEY = "routingKey";
 
-
   @Mock
   RabbitTemplate rabbitTemplate;
 
-  @Mock
+  @Spy
```

Comment:
I can't see why you can't just mock this for the standard better separation from the tests.